restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql/sms-messages.xml

```diff
@@ -72,7 +72,7 @@
                     AND sender like #{sender}
             </if>
             <if test="body != null">
-                    AND body like '%'|| #{body} ||'%'
+                    AND body like #{body}
```

Comment:
now that i am thinking
changing from `AND body like '%'|| #{body} ||'%'`
to `AND body like #{body}`
would make sql injection easier for a hacker
wdyt

can we still somehow concatenate `%` while working with sql